### DISCUSSION TITLE
如果HA不能替代中枢网关实现本地控制，那支持HA还有多大意义

### DISCUSSION THREAD

**Discussion by github180556** [XiaoMi/ha_xiaomi_home#43] [Ideas]
不如MIOT AUTO 和gateway3

**Comment by yyxida** [comment]
😅这两个不也要中枢网关才能用。你不会想是把中枢网关开源出来吧，小米又不是慈善企业。

**Comment by huhuime** [comment]
把所有本地控制的令牌全部存到HA里，你觉得小米敢这么做。第3方当然无所谓

**Comment by iKira** [comment]
HA的作用是聚合多个IoT平台的设备，不是成为米家IoT的皮。

**Comment by FDrag0n** [comment]
这标题就像“天天吃米饭就能吃饱，那么吃肉还有什么意义”一样，还是吃太饱了。你觉得不好用可以继续用MIOT AUTO啊，而不是在这里问这个项目有什么意义，开源主打一个来去自由好吧
